recipe/meta.yaml

```diff
@@ -32,9 +32,9 @@ requirements:
     # For applying patches
     - git                          # [win]
     - numpy
-    - hdf5                # [unix]
+    - hdf5                         # [unix]
     - eigen 3.2.*
-    - jasper 1.900.1                       # [unix]
+    - jasper 1.900.1               # [unix]
```

Comment:
This should be in `conda-forge-pinning` now. So we can relax this pinning and re-render.